src/main/java/com/pusher/rest/data/TriggerResult.java

```diff
@@ -0,0 +1,70 @@
+package com.pusher.rest.data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gson.Gson;
+
+import com.pusher.rest.data.Result.Status;
+
+public class TriggerResult {
+    static class Data {
+        public final Map<String, String> event_ids;
```

Comment:
No snake case. You might need to annotate or pass something to the deserialiser...